src/App.jsx

```diff
@@ -3,26 +3,47 @@ import './App.scss';
 import './styles/general.scss';
 import { TodoList } from './components/TodoList';
 import { CurrentUser } from './components/CurrentUser';
+import { getAllTodos } from './api.js';
 
 class App extends React.Component {
   state = {
     todos: [],
     selectedUserId: 0,
   };
 
+  componentDidMount() {
+    getAllTodos()
+      .then(todos => this.setState({ todos }));
+  }
+
+  selectUser = (id) => {
+    this.setState({
+      selectedUserId: id,
+    });
+  }
+
+  RemoveUser = () => {
+    this.setState({
+      selectedUserId: 0,
+    });
+  }
```

Comment:
why with capital letter??
```suggestion
  removeUser = () => {
    this.setState({
      selectedUserId: 0,
    });
  }
```